backend/db/init_db.py

```diff
@@ -0,0 +1,46 @@
+import os
+
+from csv import reader
+
+def _get_data():
+    filepath = os.path.join(os.getcwd(), 'backend/db/StarterCodeSampleData.csv')
+    with open(filepath) as data:
+        csv_reader = reader(data)
+
+        header = [column.lower() for column in next(csv_reader)]
+        return [zip(header, row) for row in csv_reader]
+
+
+def _init_firestore_db():
+    pass
+
+
+def _init_postgres_db(app):
+    from . import db
+    from .models import RecycledMaterial
+
+    # Fixes a weird SQLAlchemy issue
```

Comment:
I still feel like it isn't a SQLAlchemy issue. Lets both take a closer look into this.

Reply:
http://flask-sqlalchemy.pocoo.org/2.3/contexts/